src/main/msp/msp.c

```diff
@@ -4082,7 +4082,13 @@ static mspResult_e mspCommonProcessInCommand(mspDescriptor_t srcDesc, int16_t cm
             if ((int8_t)addr == -1) {
                 /* Set general OSD settings */
 #ifdef USE_MAX7456
```

Comment:
It seems odd there's a driver based define here... smells.

Reply:
Agreed and there were also nested `#ifdef USE_OSD` so I've tidied that up in a new commit.